assets/queries/terraform/aws/redis_not_compliant/query.rego

```diff
@@ -0,0 +1,20 @@
+package Cx
+
+CxPolicy [ result ] {
+  resource := input.document[i].resource.aws_elasticache_cluster[name] 
+  min_version_string = "4.0.10"
+  eval_version_number(resource.engine_version) <= eval_version_number(min_version_string)
+	
+    result := {
+                "documentId": 		  input.document[i].id,
+                "searchKey": 	      sprintf("resource.aws_elasticache_cluster[%s].engine_version", [name]),
+                "issueType":		  "IncorrectValue",
+                "keyExpectedValue":   sprintf("resource.aws_elasticache_cluster[%s].engine_version is compliant with the requirements", [name]),
+                "keyActualValue": 	  sprintf("resource.aws_elasticache_cluster[%s].engine_version isn't compliant with the requirements", [name])
```

Comment:
"keyExpectedValue":   sprintf("aws_elasticache_cluster[%s].engine_version is compliant with the requirements", [name]),
                "keyActualValue": 	  sprintf("aws_elasticache_cluster[%s].engine_version isn't compliant with the requirements", [name])